examples/simulator-1/client.mjs

```diff
@@ -1,28 +1,31 @@
 import fetch from 'node-fetch';
 import assert from 'assert';
+import waitPort from 'wait-port';
 
 // const fetch = require('node-fetch')
 
-const hostname = "http://localhost:3000"
+const hostname = "http://localhost"
+const port = 3001
+const address = `${hostname}:${port}`
 
 const getStates = async () => {
-  const r = fetch(`${hostname}/states`)
+  const r = fetch(`${address}/states`)
     .then(response => response.json())
     .then(data => data);
```

Comment:
I continue to think all these `.then`s should be replace with `await` style. This function, for instance, is wrong because you don't `await` the `fetch`

Reply:
oh sorry

Reply:
What do the seven `.then(data => data);` in this file do?

Is it safe to remove them?